src/tests/utils.cairo

```diff
@@ -1,40 +1,41 @@
+pub(crate) mod common;
 pub(crate) mod constants;
-pub mod foundry;
-pub use foundry::declare_and_deploy;
+pub(crate) mod events;
+pub(crate) mod signing;
+pub use common::{declare_class, declare_and_deploy, declare_and_deploy_at, deploy, deploy_at};
 
-pub use foundry::spy_on;
+pub use events::{spy_on, EventSpyExt};
+// use starknet::ContractAddress;
```

Comment:
Why are these commented? We can remove the commented functions.

Reply:
Removed